app/src/main/java/com/github/libretube/services/OnClearFromRecentService.kt

```diff
@@ -0,0 +1,33 @@
+package com.github.libretube.services
+
+import android.app.NotificationManager
+import android.app.Service
+import android.content.Intent
+import android.os.IBinder
+import androidx.core.content.getSystemService
+
+class OnClearFromRecentService : Service() {
+    private var nManager: NotificationManager? = null
+
+    override fun onBind(p0: Intent?): IBinder? {
+        return null
+    }
+
+    override fun onStartCommand(
+        intent: Intent?,
+        flags: Int,
+        startId: Int,
+    ): Int {
+        nManager = getSystemService<NotificationManager>()
+        return START_NOT_STICKY
+    }
+
+    override fun onTaskRemoved(rootIntent: Intent?) {
+        nManager?.cancel(PLAYER_NOTIFICATION_ID)
+        stopSelf()
+    }
+
+    companion object {
+        private const val PLAYER_NOTIFICATION_ID = 1
```

Comment:
That constant is already defined somewhere else (iirc in NotificationHelper.kt`)

Reply:
I used it, but in the remote debug build it cannot resolve it, i don't know why. Will I do it ?